src/engine/QueryPlanner.cpp

```diff
@@ -195,6 +195,24 @@ std::vector<SubtreePlan> QueryPlanner::createExecutionTrees(ParsedQuery& pq,
     }
   }
 
+  auto& postValues = pq.postQueryValuesClause_;
```

Comment:
Note to self: If the limit should be applied before post values, then we need to make sure it is properly applied in this case and adjust the exporter to ignore it.